tests/all/module_linking.rs

```diff
@@ -253,8 +253,8 @@ fn limit_memories() -> Result<()> {
             )
         "#,
     )?;
-    let mut store = Store::new(&engine, ());
-    store.limiter(StoreLimitsBuilder::new().memories(10).build());
+    let mut store = Store::new(&engine, StoreLimitsBuilder::new().memories(10).build());
+    store.limiter(|s| s as &mut dyn ResourceLimiter);
```

Comment:
Is the `as &mut dyn ResourceLimiter` required here in that rustc can't infer it?

Reply:
`|s| s` doesn't attempt to coerce `s` AFAIK.